### PR TITLE
Feature/237 Update 'more info' links aka Project Demo URL in bundle info

### PR DESCRIPTION
**Pull request by cathysnider**

### CODE REVIEW

File: modules/custom/cu_advanced_content_bundle/cu_advanced_content_bundle.info [hunk 1]
```diff
@@ -2,7 +2,7 @@ name = Advanced Content
 description = Add tabbed content, content lists, content grids and video reveal functionality to your site.
 core = 7.x
 package = CU-Advanced Content
-project_demo_url = http://www.colorado.edu/webcentral/node/668
+project_demo_url = https://websupport.colorado.edu/category/142-advanced-content
```
Comment (jwfuller) [inline modules/custom/cu_advanced_content_bundle/cu_advanced_content_bundle.info]: I think the overview page is more useful than the category page for a bundle. Could we use links like https://websupport.colorado.edu/article/119-advanced-content when possible?

Comment (cathysnider) [reply modules/custom/cu_advanced_content_bundle/cu_advanced_content_bundle.info]: I asked the Support Team this question. They said plz use the Categories, not Overview.

Screen Shot 2019-08-16 at 1 49 37 PM

File: modules/custom/cu_livechat_bundle/cu_livechat_bundle.info [hunk 2]
```diff
@@ -6,6 +6,7 @@ package = CU-LiveChat
 project = cu_livechat_bundle
 project status url = https://github.com/CuBoulder/cu_livechat_bundle
 project_demo_url = http://www.colorado.edu/webcentral/node/1532
+https://websupport.colorado.edu/article/64-live-chat
```
Comment (jwfuller) [inline modules/custom/cu_livechat_bundle/cu_livechat_bundle.info]: This should be a replace

Comment (jwfuller) [inline modules/custom/cu_livechat_bundle/cu_livechat_bundle.info]: This should be a replacement

Comment (cathysnider) [reply modules/custom/cu_livechat_bundle/cu_livechat_bundle.info]: Fixed in commit: https://github.com/CuBoulder/express_mono/pull/243/commits/26983e9d53baa214ba14e0d039ce7b17ef14e89d

**Pull request merged**